.chloggen/instana-exporter-https-option.yaml

```diff
@@ -0,0 +1,11 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: exporterconfig
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: The exporter config options expects the endpoint URL to be https://
```

Comment:
The exporter config options expects the endpoint URL to be https:// --> The exporter config options should be HTTPS endpoint URL only